community/kernel/src/main/java/org/neo4j/kernel/impl/index/schema/ZonedDateTimeLayout.java

```diff
@@ -84,19 +84,19 @@ public void readKey( PageCursor cursor, ZonedDateTimeSchemaKey into, int keySize
         if ( isZoneId( encodedZone ) )
         {
             into.zoneId = asZoneId( encodedZone );
-            into.zoneOffsetSeconds = 0;
+            into.zoneOffsetMinutes = 0;
         }
         else
         {
             into.zoneId = -1;
-            into.zoneOffsetSeconds = asZoneOffset( encodedZone );
+            into.zoneOffsetMinutes = asZoneOffset( encodedZone );
         }
         into.setEntityId( cursor.getLong() );
     }
 
     private int asZoneOffset( int encodedZone )
     {
-        return encodedZone;
+        return (short) encodedZone;
```

Comment:
If you should benefit from returning a `short` here shouldn't the return type of the method also be `short`?